files/kafka.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Apache Kafka
+Requires=zookeeper.service
+After=zookeeper.service
+[Service]
+Type=simple
+User=dojokafka
+Group=dojo
+ExecStart=/home/dojokafka/kafka/bin/kafka-server-start.sh /home/dojokafka/kafka/config/server.properties
```

Comment:
please use `jinja` templates, and all vars should reside in env-vars

Reply:
Working on it

Reply:
Updated templates and variables